addons/web/static/tests/fields/relational_fields_tests.js

```diff
@@ -2277,6 +2277,123 @@ QUnit.module('relational_fields', {
         });
     });
 
+    QUnit.test('pressing TAB on a \'no_quick_create\' many2one should not trigger M2ODialog', function (assert) {
```

Comment:
test name is wrong: it should open a M2O dialog
actually, i wouldn't explain what should happen in test name, but rather what is done, in this case 'pressing TAB on a 'no_quick_create' many2one.